framework/test/mocha/unit/modules/chain/blocks/verify.js

```diff
@@ -316,9 +316,9 @@ describe('blocks/verify', () => {
 	describe('verifyVersion', () => {
 		describe('when there are no exceptions for block versions', () => {
 			describe('when block height provided', () => {
-				it('should return no error when block version = 1', async () => {
+				it('should return no error when block version = 2', async () => {
```

Comment:
If exceptions are not present for block versions, doesn't it mean that we don't have block version updates at heights? If so, why do the block_version return an error?

Reply:
It returns always the current block version which is now set to `2`.